docs/framework/data/adonet/dataset-datatable-dataview/annotating-typed-datasets.md

```diff
@@ -81,7 +81,7 @@ xmlns:codegen="urn:schemas-microsoft-com:xml-msprop"
 <xs:schema id="CustomerDataSet"   
       xmlns:codegen="urn:schemas-microsoft-com:xml-msprop"  
       xmlns=""   
-      xmlns:xs="http://www.w3.org/2001/XMLSchema"   
+      xmlns:xs="https://www.w3.org/2001/XMLSchema"   
```

Comment:
this comment will apply to all w3.org links being used in code. I'd not touch those with this PR.